crates/node/src/main.rs

```diff
@@ -213,11 +215,13 @@ async fn run(config: Arc<Config>) -> Result<()> {
         });
     }
 
+    let public_node_key = PublicKey::from_secret_key(&node_key);
```

Comment:
A remark about how private keys are stored in the memory. As the run() function never returns, the `node_key` isn't drop and always stay in the node memory.  For Devnet, it's not an issue, but more generally it's something we've to take care. Usually, private key are zeroed after being used.